### PR TITLE
Add resource for invite accepter.

### PR DESCRIPTION
**Pull request by JoshiiSinfield**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -72,3 +72,14 @@ variable "threat_intel_list_path" {
   default     = ""
   type        = "string"
 }
+
+
+variable "accept_invite" {
+  description = "Accept invitation from a master account."
+  default     = false
+}
```
Comment (joseph-wortmann) [inline variables.tf:80]: Would you mind adding `type = "string"`? Not having it tickles my OCD.

File: variables.tf [hunk 2]
```diff
@@ -72,3 +72,14 @@ variable "threat_intel_list_path" {
...
+
+variable "accepter_master_account_id" {
+  description = "Account ID for Guard Duty Master. Required if accept_invite = true"
+  default = ""
```
Comment (joseph-wortmann) [inline variables.tf:84]: Same `type = "string"` here. And please put all the `=` in a straight line. My OCD again...

**Review by joseph-wortmann — CHANGES_REQUESTED** [review]
Minor requests. Looks good to me otherwise. @vchinnakotla?

File: main.tf [hunk 3]
```diff
@@ -44,3 +44,9 @@ resource "aws_guardduty_member" "members" {
   invite             = "${lookup(var.member_list[count.index], "invite")}"
   invitation_message = "please accept guardduty invitation"
 }
+
+resource "aws_guardduty_invite_accepter" "member_accepter" {
+  count             = "${var.accept_invite && (length(var.accepter_master_account_id) > 0) ? 1 : 0}"
+  detector_id       = "${aws_guardduty_detector.master.id}"
+  master_account_id = "${var.accepter_master_account_id}"
+}
```
Comment (vchinnakotla) [inline main.tf:52]: Actually this module is meant for master guardduty which can invite other member accounts. This above code changes also works, if we want to make this account to be invited from other master account.
You or I can update the readme accordingly..

File: variables.tf (hunk 1 above)
Comment (JoshiiSinfield) [reply variables.tf:80]: I can do. It was tickling mine too, but the default is string so didn't bother!

File: variables.tf (hunk 2 above)
Comment (JoshiiSinfield) [reply variables.tf:84]: will do... I thought I'd ran fmt before pushing, but missed this file.

File: main.tf (hunk 3 above)
Comment (JoshiiSinfield) [reply main.tf:52]: The module has nothing that makes it master only. You've already account for the setup of a member account's detector inherently due to the count on the member list. 

I use this module for both sides of the fence.

I'll update the README - apologies I should have done so already!

**Pull request merged**